routes/clients.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express');
 const router = express.Router();
 const clients = require('../data/clients.json');
+const formidable = require('formidable');
```

Comment:
As far as I see we are not using formidable below?

Reply:
Oh yeah because we don't have database or a cloud, one sec